app/views/developers/_form.html.erb

```diff
@@ -83,22 +83,79 @@
     <div class="bg-white shadow mt-8 px-4 py-5 lg:rounded-lg sm:p-6">
       <div class="md:grid md:grid-cols-3 md:gap-6">
         <div class="md:col-span-1">
-          <h3 class="text-lg font-medium leading-6 text-gray-900">Availability</h3>
-          <p class="mt-1 text-sm text-gray-500">If you're currently looking for work, how soon can you start?</p>
+          <h3 class="text-lg font-medium leading-6 text-gray-900">Work preferences</h3>
```

Comment:
Obviously it's totally up to you, but if you keep all of these strings in the `i18n` files, the app becomes much easier to translate into another language later down the line. 

If you are planing on taking that approach, I would recommend enabling this config in `development.rb`

```
  # Raises error for missing translations
  config.action_view.raise_on_missing_translations = true
```

This will ensure an exception is thrown (in dev) if a translation is missing.

Reply:
No, I agree. I'm slowly moving all the strings into the translation. I added #110 to keep track of this separately.

I also enabled `raise_on_missing_translations` for both development and test environments.